### PR TITLE
My solutions to lesson02 tasks

### PR DESCRIPTION
**Pull request by AlexSkrn**

### CODE REVIEW

File: students/alex_skrn/lesson02/grid.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+"""Print a 2x2 grid, define two print_grid functions."""
+# PART 1 - print a 2x2 grid, each cell 4 dashes large
+
+plus, minus, bar, space = '+', '-', '|', ' '
+numcells = 2
```
Comment (mdgregor) [inline students/alex_skrn/lesson02/grid.py]: for two worded variables, you'll want to separate them via a `_`.  So `num_cells` and `num_dashs`.  This makes them more readable and is the standard python follows.

File: students/alex_skrn/lesson02/grid.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+numdashes = 4
+
+# construct the plus-and-dashes line, + - - - - + - - - - +
+plusline = ''
+for i in range(numcells):
```
Comment (mdgregor) [inline students/alex_skrn/lesson02/grid.py]: When you aren't using the variable in the for loop, like `i` here, common usage is to use an `_` so other programmers know that the variable is a throw away.
```
for _ in range(num_cells):
```

File: students/alex_skrn/lesson02/grid.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    plusline += plus + space + (minus+space)*numdashes
+plusline += plus
+
+# construct the bars-and-spaces line, |         |         |
+barline = ''
+for i in range(numcells):
+    barline += bar + space + space*2*numdashes
```
Comment (mdgregor) [inline students/alex_skrn/lesson02/grid.py]: You'll want to get spaces between your operators.  If you want to make it more readable, don't hesitate to throw some parens around even if order of operations doesn't need it:
```
bar_line += bar + space + (space * 2 * num_dashes)
```
This adds to the readability.

**Comment by AlexSkrn** [comment]
Thank you for your comments! For nested for loops, as throwaway variables, I used a single underscore for the outer loop and a double underscore for the inner loop. I hope that's all right.

**Comment by mdgregor** [comment]
You'll want to stick to the single `_` no matter what.  Since its a throw away variable it doesn't matter if its overwritten by the secondary `for` loop.  Great job!

**Pull request merged**